<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1061</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Objectif :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Text :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_adresse">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>531</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>E-mail  :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_envoyer">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>349</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{

     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #f7f7f7, stop: 0.4 #f0f0f0,
                                 stop: 0.5 #e8e8e8, stop: 1.0 #dcdcdc);
border-radius:20px;
border-radius:15px;

}



QPushButton:hover{
background:   qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #1b83fe, stop: 0.4 #4399fe,
                                 stop: 0.5 #5da7fe, stop: 1.0 #77b5fe);
color:white;
}</string>
   </property>
   <property name="text">
    <string>Envoyer</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_objet">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>131</y>
     <width>531</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_texte">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>180</y>
     <width>531</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
